bundles/org.openhab.binding.vesync/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,341 @@
+				<required>true</required>
+				<label>Username</label>
+				<description>Name of a registered VeSync user, that allows to access the mobile application.</description>
+			</parameter>
+			<parameter name="password" type="text">
+				<context>password</context>
+				<required>true</required>
+				<label>Password</label>
+				<description>Password for the registered VeSync username, that allows to access the mobile application.</description>
+			</parameter>
+			<parameter name="backgroundDeviceDiscovery" type="boolean">
+				<label>Background Device Scans</label>
+				<description>Enable background scanning for new devices.</description>
+				<default>true</default>
+			</parameter>
+			<parameter name="airPurifierPollInterval" type="integer" min="5" step="1" unit="s">
+				<label>Air Filters/Humidifiers Poll Rate</label>
+				<description>Seconds between fetching background updates about the air purifiers / humidifiers.</description>
+				<default>60</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="AirPurifier">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+
+		<label>Air Purifier via VeSync</label>
+		<description>A Air Purifier uplinking to VeSync</description>
+
+		<channels>
+			<channel id="enabled" typeId="deviceEnabledType"/>
+			<channel id="childLock" typeId="deviceChildLockEnabledType"/>
+			<channel id="display" typeId="deviceDisplayEnabledType"/>
+			<channel id="filterLifePercentage" typeId="deviceFilterLifePercentageType"/>
+			<channel id="fanMode" typeId="airPurifierModeType"/>
+			<channel id="manualFanSpeed" typeId="airPurifierFanLevelType"/>
+			<channel id="errorCode" typeId="deviceErrorCodeType"/>
+			<channel id="airQuality" typeId="deviceAirQualityBasicType"/>
+			<channel id="airQualityPPM25" typeId="deviceAirQualityBasicType"/>
+			<channel id="configDisplayForever" typeId="deviceAFConfigDisplayForever"/>
+			<channel id="configAutoMode" typeId="deviceAFConfigAutoPrefType"/>
+			<channel id="timerExpiry" typeId="deviceAFTimerExpiry"/>
+			<channel id="configAutoRoomSize" typeId="deviceAFConfigAutoPrefRoomSizeType"/>
+			<channel id="schedulesCount" typeId="deviceAFConfigAutoScheduleCountType"/>
+			<channel id="nightLightMode" typeId="deviceAFNightLight"/>
+		</channels>
+
+		<properties>
+			<property name="Device Name"/>
+			<property name="Device Type"/>
+			<property name="MAC Id"/>
+		</properties>
+		<representation-property>macId</representation-property>
+
+		<config-description>
+			<parameter name="macId" type="text">
+				<label>MAC Id</label>
+				<description>The MAC Id of the device as reported by the API.</description>
+			</parameter>
+			<parameter name="deviceName" type="text">
+				<label>Device Name</label>
+				<description>The name allocated to the device by the app. (Must be unique if used)</description>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<thing-type id="AirHumidifier">
```

Comment:
Thing type IDs need to start with a lower case. Same for the purifier.
```suggestion
	<thing-type id="airHumidifier">
```

Reply:
Thank you 👍 The thing-type id's have been converted to lower case, code references and documentation updated.